cmd/cluster/core/create.go

```diff
@@ -118,21 +120,22 @@ type KubevirtPlatformCreateOptions struct {
 }
 
 type AWSPlatformOptions struct {
-	AWSCredentialsFile string
-	AdditionalTags     []string
-	IAMJSON            string
-	InstanceType       string
-	IssuerURL          string
-	PrivateZoneID      string
-	PublicZoneID       string
-	Region             string
-	RootVolumeIOPS     int64
-	RootVolumeSize     int64
-	RootVolumeType     string
-	EndpointAccess     string
-	Zones              []string
-	EtcdKMSKeyARN      string
-	EnableProxy        bool
+	AWSCredentialsFile      string
+	AWSCredentialSecretName string
```

Comment:
I can see where AWSCredentialSecretName is consumed?

Reply:
Good catch. This was from an earlier iteration.